cloudslang-runtime/src/test/java/io/cloudslang/lang/runtime/steps/ActionStepsTest.java

```diff
@@ -135,6 +136,75 @@ public void doJavaActionWrongClassTest() {
         Assert.assertNotNull(actionEndEvent);
     }
 
+    @Test
+    public void doActionPythonActionCheckCallArgumentsOnEvent() {
+        //prepare doAction arguments
+        RunEnvironment runEnv = new RunEnvironment();
+        ExecutionRuntimeServices runtimeServices = new ExecutionRuntimeServices();
+
+        Map<String, Serializable> callArguments = new HashMap<>();
+        callArguments.put("index", 1);
+        runEnv.putCallArguments(callArguments);
+
+        String userPythonScript = "print \"hello\"";
+
+        //invoke doAction
+        actionSteps.doAction(runEnv,new HashMap<String, Object>(), PYTHON, "", "", runtimeServices, userPythonScript, 2L, "operationName");
+
+        Collection<ScoreEvent> events = runtimeServices.getEvents();
+
+        Assert.assertFalse(events.isEmpty());
+        ScoreEvent eventActionStart = null;
+        for(ScoreEvent event:events){
+            if(event.getEventType().equals(ScoreLangConstants.EVENT_ACTION_START)){
+                eventActionStart = event;
+            }
+        }
+        assert eventActionStart != null;
```

Comment:
I am not much of an `assert` fan in unit tests -- I think you should use `assertNotNull` instead